strings/ship_names.json

```diff
@@ -465,9 +526,9 @@
 	"WEAPONS": [
 		"Aruval",
 		"Assegai",
-		"Blackjack",
```

Comment:
after googling i found out that blackjack is actually a type of weapon, but I'm not objecting to removing it as it's already in general list. Just wanted to put that fun fact out there

Reply:
Yep, that's why it was there to begin with. I'd forgotten it was also in the general list, though, and I want to avoid duplicates.